wiki/Installing-Oppia-Android.md

```diff
@@ -81,36 +84,40 @@ Please follow these steps to set up Oppia Android on your local machine.
 ## Set up and run tests
 Testing the app is an integral part of our development process. You will need to test all code changes to ensure that the app works correctly, therefore it is important to ensure that your test configuration works.
 
-We run tests in either Espresso(`app` module tests) or Robolectric(non-app module tests), meaning tests will run in either the emulator or on Gradle/Bazel via the terminal.
+Robolectric allows for faster unit testing of Android applications without the need for device or emulator setup.
+
+### Configure Robolectric Tests
```

Comment:
@MohitGupta121, these instructions were originally written for only non-app module tests, but we also now need to include instructions to run app module tests on gradle(the Junit configuration). Please include a step by step guide here to run the tests such as SplashActivityTest on Robolectic, and ensure that these are also updated on the dedicated testing page as well.
<img width="1386" alt="Screenshot 2023-10-28 at 10 41 43" src="https://github.com/oppia/oppia-android/assets/59600948/6164bae2-817b-4f17-a045-45fa87d8087a">

Reply:
Done.

Reply:
This looks great now, thanks.